packages/kit/src/views/UniversalSearch/components/SearchResultItems/UniversalSearchMarketTokenItem.tsx

```
@@ -42,14 +42,18 @@ export function UniversalSearchMarketTokenItem({
         from:
           searchStatus === ESearchStatus.init ? 'trendingList' : 'searchList',
       });
-      setTimeout(() => {
-        universalSearchActions.current.addIntoRecentSearchList({
-          id: coingeckoId,
-          text: symbol,
-          type: item.type,
-          timestamp: Date.now(),
-        });
-      }, 10);
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)